rest-server/core/src/main/scala/com/actionml/core/search/elasticsearch/ElasticSearchSupport.scala

```diff
@@ -96,12 +96,12 @@ class ElasticSearchClient private (alias: String, client: RestClient)(implicit w
       else if (version.startsWith("7.")) v7
       else {
```

Comment:
@andreyshikov assume we are using ES8, this should not be a WARN, it should be assumed to work fine.